language/diem-framework/modules/doc/AccountFreezing.md

```diff
@@ -235,7 +235,7 @@ The <code><a href="AccountFreezing.md#0x1_AccountFreezing_FreezingBit">FreezingB
     <a href="DiemTimestamp.md#0x1_DiemTimestamp_assert_genesis">DiemTimestamp::assert_genesis</a>();
     <a href="CoreAddresses.md#0x1_CoreAddresses_assert_diem_root">CoreAddresses::assert_diem_root</a>(dr_account);
     <b>assert</b>(
-        !<b>exists</b>&lt;<a href="AccountFreezing.md#0x1_AccountFreezing_FreezeEventsHolder">FreezeEventsHolder</a>&gt;(<a href="Signer.md#0x1_Signer_address_of">Signer::address_of</a>(dr_account)),
+        !<b>exists</b>&lt;<a href="AccountFreezing.md#0x1_AccountFreezing_FreezeEventsHolder">FreezeEventsHolder</a>&gt;(<a href="_address_of">Signer::address_of</a>(dr_account)),
```

Comment:
Right now, if the Move stdlib modules are excluded from the doc-gen for the Diem framework, bad links like this will be generated. We need to make a choice here and here are a few I can think of:
1. Continue including these modules as part of the Diem Framework docs. (We'll have the Move docs duplicated in both crates though.)
2. Try to fix the links and make them point to the docs in `move-stdlib`. Not sure if this is practical... It might require some additional work when pushing these docs to our website.
3. Keep the docs in `diem-framework`. Probably not a long term solution since Move will need proper documentation as a language.

Reply:
The documentation generator already has an option to find docs generated for another package in a different place. It is used for documentation of transaction scripts. For example, [here](https://github.com/diem/diem/blob/db2e9efec840e9ccbb9f0dfd8fa79cd6ba0bc9ad/language/diem-framework/src/lib.rs#L234) the path to the stdlib docs is passed when generating docs for scripts. Did you try to use this? If you run in trouble with this for this PR, I guess one way to sove it submit as is, and me/Tim/Junkil followup and fix after you merged.

Reply:
I added the path argument following your example
```
pub fn build_stdlib_doc(with_diagram: bool) {
    move_stdlib::build_doc(
        STD_LIB_DOC_DIR,
        &move_stdlib::move_stdlib_docs_full_path(), // was ""
        vec![path_in_crate(STD_LIB_DOC_TEMPLATE)
            .to_string_lossy()
            .to_string()],
        Some(
            path_in_crate(REFERENCES_DOC_TEMPLATE)
                .to_string_lossy()
                .to_string(),
        ),
        diem_stdlib_files_no_dependencies().as_slice(),
        vec![move_stdlib::move_stdlib_modules_full_path()],
        with_diagram,
    )
}
```
The generated links are correct, but they are absolute and contain my local user name and directories:
```
-<b>use</b> <a href="">0x1::Signer</a>;
+<b>use</b> <a href="/Users/vgao/libra/language/move-stdlib/docs/Signer.md#0x1_Signer">0x1::Signer</a>;
```
With that being said I have a feeling that maybe this is because the path I'm passing to the tool is an absolute path. Let me try a relative one and see if that will work. It might be a decent hack to put in before we get tool improved.

Reply:
Relative path works
```
-<b>use</b> <a href="">0x1::Signer</a>;
+<b>use</b> <a href="../../../move-stdlib/docs/Signer.md#0x1_Signer">0x1::Signer</a>;
```